srcpkgs/kitinerary/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'kitinerary'
+pkgname=kitinerary
+version=21.04.0
+revision=1
+build_style=cmake
+configure_args="-DBUILD_TESTING=OFF"
```

Comment:
Upstream may want to know about failing tests.

Reply:
It fails now because I think extended tests need a X server running.

Reply:
Did you try running the tests with xvfb-run?

Reply:
Qt do not need X server with `pre_check() {export QT_QPA_PLATFORM=offscreen;}`, and then they still fail.

Reply:
What should I do then?

Reply:
Best to work it out with upstream, things like below have potential to make people sad.

```diff
@@ -209,11 +181,7 @@
             },
             "departureDay": "2016-06-10",
             "departureTerminal": "2F",
-            "departureTime": {
-                "@type": "QDateTime",
-                "@value": "2016-06-10T13:00:00+02:00",
-                "timezone": "Europe/Paris"
-            },
+            "departureTime": "2016-06-10T13:00:00",
             "flightNumber": "8680"
         },
         "reservationNumber": "123456",
FAIL!  : ExtractorTest::testExtract(amadeus-amadeus_1.txt) Compared values are not the same
```

Reply:
https://invent.kde.org/pim/kdepim-addons/-/issues/1

Reply:
I was talking about kitinerary tests up to now, but yes, kdepim-addons has some meaningful failues too, eg.

```
 30/163 Test  #30: pimeventsplugintest ................................................***Failed    0.10 sec
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-piotr'
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-piotr'
********* Start testing of PimEventsPluginTest *********
Config: Using QtTest library 5.15.2, Qt 5.15.2 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 10.2.1 20201203), unknown unknown
PASS   : PimEventsPluginTest::initTestCase()
QWARN  : PimEventsPluginTest::testLoadEventsForDataRange() kf.calendarcore: Invalid dtStart
QWARN  : PimEventsPluginTest::testLoadEventsForDataRange() kf.calendarcore: Invalid dtStart
QWARN  : PimEventsPluginTest::testLoadEventsForDataRange() kf.calendarcore: Invalid dtStart
QWARN  : PimEventsPluginTest::testLoadEventsForDataRange() kf.calendarcore: Invalid dtStart
FAIL!  : PimEventsPluginTest::testLoadEventsForDataRange() Compared values are not the same
   Actual   (actual.size())  : 2
   Expected (expected.size()): 40
```

On dbus: runing tests under dbus-run-session like okular template does does not really help - test 2 freeze (but it passed without dbus - ???). Maybe we would need to disable dbus depending test after deciding to ignore/resolving others.